src/main/java/org/sonarqube/gradle/SonarQubePlugin.java

```diff
@@ -109,7 +109,12 @@ private static List<File> registerAndConfigureResolverTasks(Project topLevelProj
     List<File> resolverFiles = new ArrayList<>();
     topLevelProject.getAllprojects().forEach(target ->
       target.getTasks().register(SonarResolverTask.TASK_NAME, getCompatibleTaskType(GradleVersion.current()), task -> {
+        // register happen at configuration, we must delay computation to be sure configuration is finished before computing the value
```

Comment:
The commit is not super precise. The call to register happens at registration time not configuration, but the action that is given as the last argument is executed at configuration time.